content/dogepedia/articles/history-of-dogecoin.md

```diff
@@ -5,6 +5,16 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
  
-The history of Dogecoin - from Billy and Jackson's quasi-mythical launch to our times, passing through Dogecars and Jamaican bobsled teams, generosity and insanity. Picture of a shibe-historian with a long beard at the top. 
+Dogecoin was originally created as a joke in late 2013 by Billy Markus and Jackson Palmer. Both Billy and Jackson thought that cryptocurrency was being taken far too seriously and wasn’t much fun. 
 
-Consult Billy and the current developers, and read through Sporklin's post history, which is a repository on the history of Dogecoin.
\ No newline at end of file
+On December 6th, 2013, Dogecoin was launched taking influence from the famous Doge meme: An image of a very adorable Shiba Inu dog with multi-coloured comic sans text that appears as the dog’s “thought bubbles”. 
+
+This famous meme was based on a 2010 photograph of a Japanese female Shiba Inu dog called **Kabosu**, who was adopted by Japanese kindergarten teacher Atsuko Satō from an animal shelter in 2008.  
+
+Much to Billy and Jackson’s surprise, Dogecoin was a near-instant success, and gained tremendous popularity on sites such as Reddit, where it was —and still is — used as a tipping currency. When Dogecoin was just 2-weeks old, it reached more transactions per day than Bitcoin. Moreover, within the first month there were over one-million unique visitors to this website!  
+

```

Comment:
```suggestion

In 2014, both Billy and Jackson left the development efforts, while a Dogecoin Core Development team was formed. The team, comprised of a small group of core maintainers and supported by more than 40 contributors over the years, has been maintaining and developing Dogecoin since then.

```

Reply:
I think it is important to give some information about the formation of the development team and the continuity to this day.